### ISSUE TITLE
Resize Images at Read in (imread)

### ISSUE THREAD

**Issue by mh-nyris** [mseitzer/pytorch-fid#32]
Hey there,
I ran into problems as my images are of variable size.
Does it make sense to consider resizing images in imread?
Image.open(filename).resize((299, 299))
Best,
Mike

**Comment by septasset** [comment]
I've also encounted this issue, my solution is to transform before dataloader:
```
    dataset = ImagePathDataset(files,
        transforms=TF.Compose([
            TF.Resize((299,299)),
            TF.ToTensor(),
    ]))
```

**Comment by KomputerMaster64** [comment]
Thank you for the solution.
[Here is the reference architecture of the InceptionV3 model](https://pytorch.org/hub/pytorch_vision_inception_v3/)

**Comment by Newbeeer** [comment]
I observe that such resize operation would change the FID score significantly.

**Comment by kmittle** [comment]
> I've also encounted this issue, my solution is to transform before dataloader:
> 
> ```
>     dataset = ImagePathDataset(files,
>         transforms=TF.Compose([
>             TF.Resize((299,299)),
>             TF.ToTensor(),
>     ]))
> ```

The InceptionV3 model in PyTorch is trained on ImageNet. According to the official documentation, it uses the mean and standard deviation of ImageNet for normalization. However, if you directly use `ToTensor()`, it will only normalize the values to the range 0-1.

**Comment by kmittle** [comment]
> > I've also encounted this issue, my solution is to transform before dataloader:
> > ```
> >     dataset = ImagePathDataset(files,
> >         transforms=TF.Compose([
> >             TF.Resize((299,299)),
> >             TF.ToTensor(),
> >     ]))
> > ```
> 
> The InceptionV3 model in PyTorch is trained on ImageNet. According to the official documentation, it uses the mean and standard deviation of ImageNet for normalization. However, if you directly use `ToTensor()`, it will only normalize the values to the range 0-1.

https://pytorch.org/vision/stable/models/generated/torchvision.models.inception_v3.html
![image](https://github.com/mseitzer/pytorch-fid/assets/60066336/e6e991b7-732f-4f56-9540-e96f8b87c1f3)

**Comment by whale-z** [comment]
> > I've also encounted this issue, my solution is to transform before dataloader:
> > ```
> >     dataset = ImagePathDataset(files,
> >         transforms=TF.Compose([
> >             TF.Resize((299,299)),
> >             TF.ToTensor(),
> >     ]))
> > ```
> 
> The InceptionV3 model in PyTorch is trained on ImageNet. According to the official documentation, it uses the mean and standard deviation of ImageNet for normalization. However, if you directly use `ToTensor()`, it will only normalize the values to the range 0-1.

Hi, I encountered the same problem. How did you solve it?
